makecomparison.sh

```diff
@@ -32,23 +32,16 @@ echo "-----------"
 echo "The testresult is $testresult"
 
 # if the APP_ENVIRONMENT is staging AND
-#     if tests are successfull and you find the text [AUTO-PROCEED] in the commit message,
-#      trigger the workflow 'release-finish'
-#      else trigger the workflow 'release-hold-and-finish'
-if [ "$APP_ENVIRONMENT" = "staging" ]; then
-  if [ $testresult = 0 ]  && [[ $git_message == *"[AUTO-PROCEED]"* ]]; then
-    echo "The auto-proceed message exists and the testresult is 0"
-    #Parameter 3 is release_init
-    #Parameter 4 is release_hold
-    #Parameter 5 is release_finish
-    ./trigger_api.sh "$CIRCLE_PROJECT_REPONAME" "$CIRCLE_BRANCH" false false true
+# if tests are successfull and there is no [HOLD] in the commit message,
+# mark job as successfull
+if [ "$APP_ENVIRONMENT" = 'staging' ]; then
+  if [ $testresult = 0 ]  && [[ $git_message != *"[HOLD]"* ]]; then
+    echo "No hold message exists and the testresult is 0"
+    exit 0
   else
-    echo "Either the auto-proceed does not exist or the testresult is not 0"
-    #Parameter 3 is release_init
-    #Parameter 4 is release_hold
-    #Parameter 5 is release_finish
-    ./trigger_api.sh "$CIRCLE_PROJECT_REPONAME" "$CIRCLE_BRANCH" false true false
+    echo "Either the hold message exists or the testresult is not 0"
```

Comment:
Could be helpful if we echo which one of the 2 is causing the job to fail

Reply:
I amended the commit to simplify the code. Actually we don't need to check for the tests here, we already check that a bit earlier and we use the exit code at the end of script. We only need to check for a `[HOLD]` prefix and exit early.